### PR TITLE
Fluxion related

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/polynomial/regular/impl/laguerre.hpp [hunk 1]
```diff
@@ -88,62 +88,37 @@ laguerre_(EVE_SUPPORTS(cpu_), successor_type const&, N n, L l, T x, T pl, T plm1
...
-          auto p = p0;
-          p0     = p1;
-          p1     = if_else(test, successor(laguerre)(c, m, x, p0, p), p1);
-          c      = inc(c);
-          test   = c < n;
-        }
-        return if_else(iseqzn, one, if_else(iseqzm, laguerre(n, x), p1));
+      auto n = convert(nn, as<elt_t>());
+      auto c = one(as(n));
+      ;
```
Comment (jfalcou) [inline include/eve/module/polynomial/regular/impl/laguerre.hpp:110]: spurriosu ;

File: include/eve/module/polynomial/regular/laguerre.hpp [hunk 2]
```diff
@@ -71,8 +71,7 @@ namespace eve
 //!     The expression `successor(laguerre)(n, m, x, ln, lnmm1)` implements the three term
 //!     recurrence relation for the associated Laguerre polynomials,
 //!     \f$\displaystyle \mbox{L}_{n+1}^m =
-//!     \left((m+2n+1-x)\mbox{L}_{n}^{m}-(m+n)\mbox{L}_{n-1}^{m}\right)/(n+1)\f$ The expression
```
Comment (jfalcou) [inline include/eve/module/polynomial/regular/laguerre.hpp:74]: why the removal of this doc ?

File: include/eve/module/polynomial/regular/impl/laguerre.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/polynomial/regular/impl/laguerre.hpp:110]: removed

File: include/eve/module/polynomial/regular/laguerre.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/polynomial/regular/laguerre.hpp:74]: an error

Comment (jfalcou) [reply include/eve/module/polynomial/regular/laguerre.hpp:74]: is doesn't look fixed

**Pull request merged**